kubemarine/plugins/nginx_ingress.py

```diff
@@ -280,6 +282,15 @@ def enrich_job_ingress_nginx_admission_patch(self, manifest: Manifest):
         self.enrich_image_for_container(manifest, key,
             plugin_service='webhook', container_name='patch', is_init_container=False)
 
+    def enrich_service_ingress_nginx_controller(self, manifest: Manifest):
```

Comment:
There is an opportunity to add unit test for this new enrichment method.

Reply:
Okay